openpype/hosts/nuke/api/pipeline.py

```diff
@@ -101,6 +101,12 @@ def workfile_has_unsaved_changes(self):
     def get_workfile_extensions(self):
         return file_extensions()
 
+    def get_workfile_build_placeholder_plugins(self):
```

Comment:
If not placed as a class attribute, this function https://github.com/ynput/OpenPype/blob/develop/openpype/pipeline/workfile/workfile_template_builder.py#L184 was returning empty for NukeHost

Reply:
I'm not quite sure how if this functionality ever worked on this PR https://github.com/ynput/OpenPype/pull/4266. @jakubjezek001 ?

Reply:
It did. There was not yet `NukeHost` defined.

Reply:
oh ok, thank you!!